examples/streaming/kafka fails to run when submitted in UI

[Story](https://www.pivotaltracker.com/story/show/99844130)
log info will be provided

Output from log gearpump-ui-11218@ip-10-10-10-33.log is 

```
2015-07-25 10:22:17,965 INFO Util$: Starting executor process java org.apache.gearpump.cluster.main.AppSubmitter -jar /tmp/gearpump_userapp_4493259458436363179 ; options: -Dgearpump.cluster.masters.0=ip-10-10-10-40.us-west-2.compute.internal:3000 -Dgearpump.hostname=ip-10-10-10-33
2015-07-25 10:22:18,401 INFO redirect: SLF4J: Class path contains multiple SLF4J bindings.
2015-07-25 10:22:18,402 INFO redirect: SLF4J: Found binding in [jar:file:/data1/hadoop/yarn/nm/usercache/ec2-user/appcache/application_1435832450558_0233/filecache/11/gearpump-pack-0.4.1-SNAPSHOT.tar.gz/gearpump-pack-0.4.1-SNAPSHOT/lib/slf4j-log4j12-1.7.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2015-07-25 10:22:18,402 INFO redirect: SLF4J: Found binding in [jar:file:/opt/cloudera/parcels/CDH-5.4.2-1.cdh5.4.2.p0.2/jars/slf4j-log4j12-1.7.5.jar!/org/slf4j/impl/StaticLoggerBinder.class]
2015-07-25 10:22:18,402 INFO redirect: SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
2015-07-25 10:22:18,409 INFO redirect: SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
2015-07-25 10:22:19,039 INFO redirect: Warning: get unknown argument , maybe it is a main class
2015-07-25 10:22:19,193 INFO redirect: Warning: get unknown argument , maybe it is a main class
2015-07-25 10:22:19,943 INFO redirect: Help: Submit an application to Master by providing a jar
2015-07-25 10:22:19,946 INFO redirect: org.apache.gearpump.cluster.main.AppSubmitter -namePrefix <application name prefix> -jar <application>.jar
2015-07-25 10:22:19,946 INFO redirect: -namePrefix  (required:false, default:)
2015-07-25 10:22:19,947 INFO redirect: -jar  (required:true)
2015-07-25 10:22:19,947 INFO redirect: Exception in thread "main" java.lang.reflect.InvocationTargetException
2015-07-25 10:22:19,948 INFO redirect:  at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2015-07-25 10:22:19,948 INFO redirect:  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
2015-07-25 10:22:19,948 INFO redirect:  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2015-07-25 10:22:19,948 INFO redirect:  at java.lang.reflect.Method.invoke(Method.java:606)
2015-07-25 10:22:19,948 INFO redirect:  at org.apache.gearpump.cluster.main.AppSubmitter$.main(AppSubmitter.scala:75)
2015-07-25 10:22:19,948 INFO redirect:  at org.apache.gearpump.util.AkkaApp$$anonfun$main$1.apply(AkkaApp.scala:22)
2015-07-25 10:22:19,949 INFO redirect:  at org.apache.gearpump.util.AkkaApp$$anonfun$main$1.apply(AkkaApp.scala:22)
2015-07-25 10:22:19,949 INFO redirect:  at scala.util.Try$.apply(Try.scala:191)
2015-07-25 10:22:19,949 INFO redirect:  at org.apache.gearpump.util.AkkaApp$class.main(AkkaApp.scala:21)
2015-07-25 10:22:19,949 INFO redirect:  at org.apache.gearpump.cluster.main.AppSubmitter$.main(AppSubmitter.scala:29)
2015-07-25 10:22:19,949 INFO redirect:  at org.apache.gearpump.cluster.main.AppSubmitter.main(AppSubmitter.scala)
2015-07-25 10:22:19,950 INFO redirect: Caused by: java.lang.NoClassDefFoundError: org/apache/gearpump/streaming/kafka/KafkaStorageFactory
2015-07-25 10:22:19,950 INFO redirect:  at org.apache.gearpump.streaming.examples.kafka.wordcount.KafkaWordCount$.application(KafkaWordCount.scala:52)
2015-07-25 10:22:19,950 INFO redirect:  at org.apache.gearpump.streaming.examples.kafka.wordcount.KafkaWordCount$.main(KafkaWordCount.scala:67)
2015-07-25 10:22:19,950 INFO redirect:  at org.apache.gearpump.util.AkkaApp$$anonfun$main$1.apply(AkkaApp.scala:22)
2015-07-25 10:22:19,950 INFO redirect:  at org.apache.gearpump.util.AkkaApp$$anonfun$main$1.apply(AkkaApp.scala:22)
2015-07-25 10:22:19,951 INFO redirect:  at scala.util.Try$.apply(Try.scala:191)
2015-07-25 10:22:19,951 INFO redirect:  at org.apache.gearpump.util.AkkaApp$class.main(AkkaApp.scala:21)
2015-07-25 10:22:19,951 INFO redirect:  at org.apache.gearpump.streaming.examples.kafka.wordcount.KafkaWordCount$.main(KafkaWordCount.scala:34)
2015-07-25 10:22:19,951 INFO redirect:  at org.apache.gearpump.streaming.examples.kafka.wordcount.KafkaWordCount.main(KafkaWordCount.scala)
2015-07-25 10:22:19,951 INFO redirect:  ... 11 more
2015-07-25 10:22:19,951 INFO redirect: Caused by: java.lang.ClassNotFoundException: org.apache.gearpump.streaming.kafka.KafkaStorageFactory
2015-07-25 10:22:19,952 INFO redirect:  at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
```

--- a/project/Build.scala
+++ b/project/Build.scala
@@ -170,7 +170,7 @@ object Build extends sbt.Build {
      libraryDependencies <+= (scalaVersion)("org.scala-lang" % "scala-reflect" % _),
      libraryDependencies ++= (
         if (scalaVersion.value.startsWith("2.10")) List("org.scalamacros" %% "quasiquotes" % "2.1.0-M5")
-        else Nil
+        else List("org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4")
       )
   )
 
@@ -258,7 +258,7 @@ object Build extends sbt.Build {
           ("org.apache.kafka" %% "kafka" % kafkaVersion classifier("test")) % "test"
         )
       )
-  ) dependsOn (streaming % "provided", dsl)
+  ) dependsOn (streaming % "provided", dsl % "provided")
 
   lazy val examples_kafka = Project(
     id = "gearpump-examples-kafka",
@@ -273,7 +273,7 @@ object Build extends sbt.Build {
         ),
         target in assembly := baseDirectory.value.getParentFile.getParentFile / "target" / scalaVersionMajor
       )
-  ) dependsOn(streaming % "test->test", streaming % "provided", external_kafka  % "test->test; provided")
+  ) dependsOn(streaming % "test->test; provided", external_kafka)
 
   lazy val fsio = Project(
     id = "gearpump-examples-fsio",

